### PR TITLE
[v6r16] small fixes

### PR DESCRIPTION
**Pull request by zmathe**

### CODE REVIEW

File: ConfigurationSystem/Client/Utilities.py [hunk 1]
```diff
@@ -609,6 +613,8 @@ def getElasticDBParameters( fullname ):
       dbPassword = result['Value']
   else:
     dbPassword = result['Value']
-  parameters[ 'Password' ] = dbPassword
+  
+  if dbPassword:
+    parameters[ 'Password' ] = dbPassword
 
   return S_OK( parameters )
```
Comment (fstagni) [inline ConfigurationSystem/Client/Utilities.py]: Just double checking: you may end up that here you return parameters that contain no user, and/or no password. Is this what you want?

Comment (zmathe) [inline ConfigurationSystem/Client/Utilities.py]: yes

**Comment by fstagni** [comment]
I don't get why the TimeLeft test is again problematic. I will look at it later.

File: MonitoringSystem/Client/ServerUtils.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+def getMonitoringDB():
+  serverName = 'Monitoring/Monitoring'
+  MonitoringDB = None
+  try:
+    from DIRAC.MonitoringSystem.DB.MonitoringDB import MonitoringDB
+  except:
+    pass
+  return getDBOrClient( MonitoringDB, serverName )
+
+monitoringDB = getMonitoringDB()
```
Comment (fstagni) [inline MonitoringSystem/Client/ServerUtils.py]: If this is a singleton, then this should be called gMonitoringDB

File: MonitoringSystem/Client/ServerUtils.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  gLogger.info( 'Can not connect to DB will use %s' % serverName )
+  return RPCClient( serverName )
+
+def getMonitoringDB():
+  serverName = 'Monitoring/Monitoring'
+  MonitoringDB = None
+  try:
+    from DIRAC.MonitoringSystem.DB.MonitoringDB import MonitoringDB
+  except:
```
Comment (fstagni) [inline MonitoringSystem/Client/ServerUtils.py]: ImportError is the only thing that can happen here.

File: MonitoringSystem/Client/ServerUtils.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+def getDBOrClient( DB, serverName ):
+  """ Tries to instantiate the DB object and returns it if we manage to connect to the DB, otherwise returns a Client of the server
+  """
+  from DIRAC import gLogger
+  from DIRAC.Core.DISET.RPCClient                            import RPCClient
+  try:
+    database = DB()
+    if database._connected:
+      return database
+  except:
```
Comment (fstagni) [inline MonitoringSystem/Client/ServerUtils.py]: `TypeError` and `AttributeError` I believe.

**Pull request merged**